aries_vcx/src/errors/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod mapping_vdrtools;
+mod mapping_agency_client;
+mod mapping_messages;
+mod mapping_others;
+pub mod error;
```

Comment:
there are some more error mappings within the credx_anoncreds and indy_vdr_ledger - maybe they should be moved into one of these mapping modules?

Reply:
oh nice! Totally missed that, moving it